relayer/relays/execution/main.go

```diff
@@ -169,12 +172,13 @@ func (r *Relay) Start(ctx context.Context, eg *errgroup.Group) error {
 				}).Info("Found events for nonce")
 
 				for _, ev := range events {
-					err := r.waitAndSend(ctx, ev)
+					err := r.waitAndSendWithRetry(ctx, ev)
 					if errors.Is(err, header.ErrBeaconHeaderNotFinalized) {
 						log.WithField("nonce", ev.Nonce).Info("beacon header not finalized yet")
 						continue
 					} else if err != nil {
-						return fmt.Errorf("submit event: %w", err)
+						log.WithFields(log.Fields{"nonce": ev.Nonce, "error": err}).Warn("submit event: message was not processed")
+						continue
```

Comment:
@ParthDesai changed to log on error only, instead of failing (which restarts the relayer).